### PR TITLE
Clean some code

### PR DESCRIPTION
**Pull request by MatthieuHernandez**

### CODE REVIEW

File: tests/dataset_tests/CIFAR-10/Cifar10Test.cpp [hunk 1]
```diff
@@ -42,13 +42,13 @@ TEST_F(Cifar10Test, trainNeuralNetwork)
 {
     StraightforwardNeuralNetwork neuralNetwork({
         Input(3072),
-        FullyConnected(100),
-        FullyConnected(25),
-        FullyConnected(10)
+        FullyConnected(80),
+        FullyConnected(30),
+        FullyConnected(10, activation::identity, Softmax())
```
Comment (Johansmm) [inline tests/dataset_tests/CIFAR-10/Cifar10Test.cpp:47]: Why do you need as activation the identity ?

File: tests/dataset_tests/CIFAR-10/Cifar10Test.cpp [hunk 2]
```diff
@@ -61,7 +61,7 @@ TEST_F(Cifar10Test, DISABLED_trainBestNeuralNetwork)
         MaxPooling(2),
         FullyConnected(128),
         FullyConnected(10, activation::identity, Softmax())
-        },
+    },
```
Comment (Johansmm) [inline tests/dataset_tests/CIFAR-10/Cifar10Test.cpp:64]: You can make this change in the first commit

**Review by Johansmm — DISMISSED** [review]

**Comment by MatthieuHernandez** [comment]
I removed the `multipleLayersNeuralNetwork` because it was too inconsistent and not really useful.

File: tests/dataset_tests/CIFAR-10/Cifar10Test.cpp (hunk 1 above)
Comment (MatthieuHernandez) [reply tests/dataset_tests/CIFAR-10/Cifar10Test.cpp:47]: Yes but we don't care. (We're not at work)

File: tests/dataset_tests/CIFAR-10/Cifar10Test.cpp (hunk 2 above)
Comment (MatthieuHernandez) [reply tests/dataset_tests/CIFAR-10/Cifar10Test.cpp:64]: Yes but we don't care. (We're not at work)

**Review by Johansmm — APPROVED** [review]

**Pull request merged**